build.sbt

```diff
@@ -23,10 +23,8 @@ libraryDependencies ++= Seq(
   "org.slf4j" % "log4j-over-slf4j" % "2.0.12", //  log4j-over-slf4j provides `org.apache.log4j.MDC`, which is dynamically loaded by the Lambda runtime
   "ch.qos.logback" % "logback-classic" % "1.5.3",
   "com.lihaoyi" %% "upickle" % "3.2.0",
-
   "com.madgag" %% "scala-collection-plus" % "0.11",
   "com.google.http-client" % "google-http-client-gson" % "1.44.1",
```

Comment:
Can we also remove this other Google library?

Reply:
yes